common/network-yarn/pom.xml

```diff
@@ -113,6 +116,13 @@
                 <include>io.netty.**</include>
               </includes>
             </relocation>
+            <relocation>
+              <pattern>com.google.common</pattern>
```

Comment:
I have changed it to use inheritance. They looked equivalent but on second thought inheritance is better so any changes to parent pom trickle down automatically.